modules/openy_features/openy_prgf/modules/openy_prgf_event_latest/config/install/field.field.paragraph.upcoming_events.field_prgf_title.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgf_title
+    - paragraphs.paragraphs_type.upcoming_events
+  module:
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
+    label: field_prgf_title
+id: paragraph.upcoming_events.field_prgf_title
+field_name: field_prgf_title
+entity_type: paragraph
+bundle: upcoming_events
+label: Title
```

Comment:
Provide a description for a user like housekeeper will be using OpenY, please. There is a logic like if you do not provide a title - it won't show title at all.